libraries/ms-drivers/inc/drv120_relay.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+// Driver for the DRV120 relay
+// Requires GPIO and status to be initialized
```

Comment:
`status` doesn't need to be initialized, so we shouldn't say it has to be initialized.